src/dp_port.c

```diff
@@ -519,11 +521,17 @@ int dp_port_meter_config(struct dp_port *port, uint64_t total_flow_rate_cap, uin
 		return DP_ERROR;
 	}
 
-	if (DP_FAILED(dp_port_total_flow_meter_config(port, total_flow_rate_cap))) {
-		DPS_LOG_ERR("Cannot set total flow meter", DP_LOG_PORT(port));
-		return DP_ERROR;
+	err = dp_port_total_flow_meter_config(port, total_flow_rate_cap);
+	if (DP_FAILED(err)) {
+		if (err == DP_GRPC_ERR_NOT_FOUND)
+			DPS_LOG_WARNING("Cannot find sysfs path or file to regulate traffic rate, thus total flow rate metering is ignored", DP_LOG_PORT(port));
```

Comment:
Unless specifically asking for `-ENOENT` This error should not mention the reason of failure, just that metering is ignored. Because currently (assuming we check `fopen`) this can also fail due to file access permissions, etc.

Reply:
you are right regarding other failure due to access permissions. and I would like to explicitly show that part of metering configuration cannot be done until we a solution is found. thus, I used `access` to explicitly to detect the existence of this sysfile, and return an -ENOENT when needed.